stdlib/json.gr

```diff
@@ -2060,7 +2073,7 @@ and parseObject = (parserState: JsonParserState) => {
  * Parses JSON string into a `Json` data structure.
  *
  * @param str: The JSON text string
- * @returns A `Result` object with either the parsed `JSON` object or an error.
+ * @returns `Ok(json)` containing the parsed data structure on a successful parse or `Err(err)` containing a parse error otherwise.
```

Comment:
No period in returns

Reply:
```suggestion
 * @returns `Ok(json)` containing the parsed data structure on a successful parse or `Err(err)` containing a parse error otherwise
```